### DISCUSSION TITLE
Kotlin coroutine problem.

### DISCUSSION THREAD

**Discussion by PritamP20** [community/community#35205] [Other Feature Feedback, Questions, & Ideas] [answered]
I tried to use coroutine and there was a problem. My app crashed. I don't know what's the problem.

**Comment by PGautam27** [comment, marked as answer]
So apparently, if you don't mention on which thread the coroutine needs to perform. Then there's a probability that the app would crash. That's considered if you have too many functions running on the same thread. 
You can handle this by **Dispatchers** . Either use **Dispatchers.IO** mostly this should work.
